langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockTitanEmbeddingModel.java

```diff
@@ -26,12 +37,40 @@ protected String getModelId() {
         return model;
     }
 
+    /**
+     * 1024 is default size of output vector for Titan Embedding model V2.
+     */
+    @Builder.Default
+    private final Integer dimensions = 1024;
```

Comment:
I would avoid setting default values, they are already defined on AWS side, right?

Reply:
Yes, correct. Good point. I'll remove it.